CONTRIBUTING.md

```diff
@@ -0,0 +1,12 @@
+# Contributing
```

Comment:
These changes are great! I think since you've sent this PR out there have been changes to the README to make it clearer.

Because of that, I think it may make the most sense to do the following:
1) Pull in changes from head
2) Remove the contribution file (since the README covers most of this) and any other changes to the README file